sim/components/firmwarevalidator/FirmwareValidator.cpp

```diff
@@ -19,3 +19,5 @@ void FirmwareValidator::Validate() {
 void FirmwareValidator::Reset() {
 //  NVIC_SystemReset();
 }
+
+void FirmwareValidator::AutoValidate() {}
```

Comment:
I Mike newlines at the enf of a file to prevent the red stop signs in the diff viewer of github

```suggestion
void FirmwareValidator::AutoValidate() {}

```

Reply:
Done